### PR TITLE
Add cross-repository-check message for 0 value

### PR DESCRIPTION
**Pull request by viktor-yakubiv**
A slightly modified commit from #336.

### CODE REVIEW

File: templates/deposit-compliance/cards/cross-repository-check-card.jsx [hunk 1]
```diff
@@ -4,29 +4,44 @@ import { Card } from 'design'
...
+        {texts.crossRepositoryCheck.download}
+      </ExportButton>
+    </>
+  )
+}
 
 const CrossRepositoryCheckCard = ({
   crossDepositLag,
-  crossDepositLagCsvUrL,
+  crossDepositLagCsvUrL: crossDepositLagCsvUrl,
```
Comment (viktor-yakubiv) [inline templates/deposit-compliance/cards/cross-repository-check-card.jsx:29]: @Joozty it seems you made a mistake typing fast. I made a quick fix renaming it in a single file. It's not a big deal but feel free to fix it if this looks annoying to you 🙂

Comment (Joozty) [reply templates/deposit-compliance/cards/cross-repository-check-card.jsx:29]: yeah, thanks for noticing. Will rename it.

Comment (Joozty) [reply templates/deposit-compliance/cards/cross-repository-check-card.jsx:29]: actually do you mind doing it here? It's should be so hard.

File: texts/depositing/cross-repository-check/failure.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Some out of {{ nonCompliantCount }} your non-compliant records might have
+been deposited in another repository. However, we could find any duplicated
```
Comment (Joozty) [inline texts/depositing/cross-repository-check/failure.md:2]:
```suggestion
been deposited in another repository. However, we could not find any duplicated
```

File: templates/deposit-compliance/cards/cross-repository-check-card.jsx (hunk 1 above)
Comment (viktor-yakubiv) [reply templates/deposit-compliance/cards/cross-repository-check-card.jsx:29]: I created #341 but I would prefer to merge this one and then check #341.

File: texts/depositing/cross-repository-check/failure.md (hunk 2 above)
Comment (viktor-yakubiv) [reply texts/depositing/cross-repository-check/failure.md:2]: Ah, yeah. Thank you!

**Pull request merged**